### ISSUE TITLE
Standalone version OS X

### ISSUE THREAD

**Issue by kirkmarkarian** [synesthesiam/opentts#16]
Sure would be cool for us non-coders to be able to use something besides MaryTTS' standalone from 2016.  Can't stand the Mac OS X voices, especially since they are not available for commercial usage.

**Comment by synesthesiam** [comment]
Have you tried running this under Docker on OS X yet? I don't have a Mac to test.

**Comment by kirkmarkarian** [comment]
No, I’ve never figured out Docker in a useable fashion.  It’s not my forte, unfortunately.  I’ll have to try again one day.
